tcms/xmlrpc/api/testcaserun.py

```diff
@@ -9,7 +9,7 @@
 from tcms.core.contrib.linkreference.views import create_link
```

Comment:
Rename file to `api/testexecution.py`

For all methods in this file, add a second decorator of the form `@rpc_method(name='TestExecution.....method_name')` to provide a forward&backwards compatible naming.